server/src/main/java/org/elasticsearch/action/admin/indices/rollover/LazyRolloverAction.java

```diff
@@ -98,7 +98,8 @@ protected void masterOperation(
                 clusterState,
                 rolloverRequest.getRolloverTarget(),
                 rolloverRequest.getNewIndexName(),
-                rolloverRequest.getCreateIndexRequest()
+                rolloverRequest.getCreateIndexRequest(),
+                rolloverRequest.indicesOptions().failureStoreOptions().includeFailureIndices()
```

Comment:
Does this mean we can lazy rollover the failure store? If not, should we explicitly pass false here and restrict setting the failure store option on rollover if the lazy setting is also specified?

Reply:
Good catch - restricted setting `failure_store` and `lazy` together (for now).